batch/batch/driver/instance_collection/pool.py

```diff
@@ -28,6 +29,8 @@
 
 log = logging.getLogger('pool')
 
+SCHEDULING_LOOP_RUNS = pc.Counter('scheduling_loop_runs', '')
```

Comment:
Hmm. Should this be per-pool or something?

Reply:
Good point, I'll add a label with the pool name